src/CaptureClient/CaptureSaveToFileProcessorTest.cpp

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <absl/container/flat_hash_set.h>
+#include <gtest/gtest.h>
+#include <stdint.h>
+
+#include <vector>
+
+#include "CaptureClient/CaptureEventProcessor.h"
+#include "CaptureFile/CaptureFile.h"
+#include "OrbitBase/Result.h"
+#include "OrbitBase/TemporaryFile.h"
+#include "OrbitBase/TestUtils.h"
+
+namespace orbit_capture_client {
+
+using orbit_base::HasValue;
+using orbit_base::TemporaryFile;
+using orbit_capture_file::CaptureFile;
+using orbit_client_protos::UserDefinedCaptureInfo;
+using orbit_grpc_protos::CaptureFinished;
+using orbit_grpc_protos::ClientCaptureEvent;
+
+static ClientCaptureEvent CreateInternedStringEvent(uint64_t key, const char* intern) {
+  ClientCaptureEvent event;
+  event.mutable_interned_string()->set_key(key);
+  event.mutable_interned_string()->set_intern(intern);
+
+  return event;
+}
+
+static ClientCaptureEvent CreateCaptureFinishedEvent() {
+  ClientCaptureEvent event;
+  event.mutable_capture_finished()->set_status(CaptureFinished::kSuccessful);
+  return event;
+}
+
+TEST(CaptureSaveToFileProcessor, SaveAndLoadSimpleCaptureWithFrameTracks) {
```

Comment:
Rename the tests from `CaptureSaveToFileProcessor` to the new class name.

Reply:
done.